docs/index.md

```diff
@@ -218,39 +218,40 @@ Typically, it's possible to keep track of this information using your monitoring
 
 Pulse uses these reports to calculate the metrics [Median time to recovery](metrics.md#median-time-to-recover) and [Change failure rate](metrics.md#change-failure-rate).
 
-#### Integrations
 
-- [Integrating with PagerDuty](pagerduty.md)
+#### Using PagerDuty
 
-- Use the Pulse CLI
+If you're using PagerDuty as an incident management tool, [use our PagerDuty integration](one-click-integrations.md#pagerduty) to send the necessary incident data to Pulse.
 
-    Send the following information when reporting **incidents** to Pulse:
+#### Using the Pulse CLI
 
-    | Field             | Description                                                | Format                                       |
-    | ----------------- | ---------------------------------------------------------- | -------------------------------------------- |
-    | identifier        | A unique identifier of the incident.                       | String                                       |
-    | timestampCreated  | Time when the incident started or was detected.            | Number<br/>(Unix epoch timestamp in seconds) |
-    | timestampResolved | Time when the incident was resolved.                       | Number<br/>(Unix epoch timestamp in seconds) |
-    | system            | Optional. Repository or component to assign to this event. | String                                       |
+Send the following information when reporting **incidents** to Pulse:
```

Comment:
As of now, this "fork" where the user decides if they're going to use the PagerDuty integration or the Pulse CLI is not very smooth.

Once we also have the GitHub integration, I will refactor this page to make the alternative between using third-party tool integrations and the Pulse CLI clearer.